src/components/graph/menus/modification-list-item.js

```diff
@@ -93,7 +91,7 @@ export const ModificationListItem = ({
             function getVoltageLevelLabel(vlID) {
                 if (!vlID) return '';
                 const vl = network.getVoltageLevel(vlID);
-                if (vl) return vl[useName ? 'name' : 'id'] || vlID;
+                if (vl) return getNameOrId(vl['name'], vl['id']) ?? vlID;
```

Comment:
remove last part ` ?? vlID` it's done by your function, and use the same syntax than elsewhere :
`getNameOrId(vl.name, vl.id)`